rest_server/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [ -z "${1}" ]; then
+  MODE="DEBUG"
+else
+  MODE="${1}"
+fi
+
+if [ -z "${2}" ]; then
+  OUTPUT_DIRECTORY_PATH="/tmp"
+else
+  OUTPUT_DIRECTORY_PATH="${2}"
+fi
+
+if [ -z "${3}" ]; then
+  GO_BIN_DIR="/usr/local/go/bin"
+else
+  GO_BIN_DIR="${3}"
+fi
+
+export PATH=${PATH}:${GO_BIN_DIR}
```

Comment:
... and each time you run it, you will be appending your GO path to PATH, again and again, in the same shell...
Can go be run w/o adding to path - like just `/usr/local/go/bin/go`? It would be more preferred.